examples/slash_commands/main.go

```diff
@@ -323,10 +320,11 @@ var (
 				s.InteractionResponseDelete(s.State.User.ID, i.Interaction)
 			})
 		},
+
+		// Interaction tokens are valid for 15 minutes and can be used to send any amount of followup messages.
+		// Followup Messages allow you send or edit responses after an initial response.
```

Comment:
I think it would be better to change wording here a little. Currently it's very confusing, especially for a beginner.
```suggestion
		// Followup messages allow for handling additional interaction related messages after the initial response.
```